src/com/xilinx/rapidwright/tests/CodePerfTracker.java

```diff
@@ -280,6 +281,15 @@ public boolean isUsingGCCallsToTrackMemory() {
      */
     public void useGCToTrackMemory(boolean useGCCalls) {
         this.trackMemoryUsingGC = useGCCalls;
+        setTrackOSMemUsage(true);
```

Comment:
Shall we leave a comment that this implies `setTrackOSMemUsage(true)`?
Is it also intentional that even a `useGCToTrackMemory(false)` implies this?